packages/react-components/src/organisms/MainNavigation.tsx

```diff
@@ -59,7 +59,7 @@ const MainNavigation: React.FC = () => (
   <nav css={containerStyles}>
     <ul css={listStyles}>
       <li>
-        <NavigationLink href={network({}).teams({}).$} icon={networkIcon}>
```

Comment:
I think there are a couple more of these, e.g. in stories or so. Can you search the codebase for `network({}).teams({}).$` and replace it? (Except of course where it's *actually* meant to link directly to teams)

Reply:
done

Reply:
You overshot the goal a bit, some of them were in places that are relying on deeply linking into the teams tab :D let me see I'll add/remove some

Reply:
Yeah looks like literally just the one in a test that was too much though :+1: